go.mod

```diff
@@ -3,22 +3,23 @@ module github.com/sneller-inc/sneller
 go 1.19
 
 require (
+	github.com/SnellerInc/sneller v0.0.0-20230505151417-5806cd3a42c7
 	github.com/grafana/grafana-plugin-sdk-go v0.159.0
+	github.com/patrickmn/go-cache v2.1.0+incompatible
```

Comment:
Do we really want to use an unmainted Go package that hasn't been updated in 4 years?

Reply:
From what I read, it's still a solid module. Most actively maintained caching modules (e.g. BigCache) are overkill for this simple usecase.